packages/web-components/src/components/va-breadcrumbs/va-breadcrumbs.tsx

```diff
@@ -25,7 +28,35 @@ export class VaBreadcrumbs {
    * Adds an aria-label attribute to the <nav /> element.
    */
   @Prop() label?: string = 'Breadcrumb';
-
+  /**
+     * Whether or not the component will use USWDS v3 styling.
+     */
+  @Prop() uswds?: boolean = false;
+  /**
+   *  Whether or not the component will wrap the breadcrumbs. This prop is available when `uswds` is set to `true`.
+   */
+  @Prop() wrapping?: boolean = false;
+  /**
+   *  Represents a list of breadcrumbs. Use an array of objects with label and href properties, and then use JSON.stringify() to convert to a string. This prop is available when uswds is set to true.
+   */
+  @Prop() breadcrumbList?: any;
```

Comment:
In the other `uswds` only prop, you wrote the last sentence with code markdown. Can you do the same thing with this one? Just looking for consistency on Storybook.

"This prop is available when `uswds` is set to `true`."